.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       DEPLOYER_PRIVATE_KEY: ${{ secrets.DEPLOYER_PRIVATE_KEY }}
       MAINNET_RPC_URL: ${{ secrets.MAINNET_RPC_URL }}
       PROPOSER_PRIVATE_KEY: ${{ secrets.PROPOSER_PRIVATE_KEY }}
+      FOUNDRY_FUZZ_SEED: 58636
```

Comment:
Wdyt of this approach so we still get some dynamic fuzzing here? https://twitter.com/PaulRBerg/status/1611116650664796166

Reply:
Done!